documentation/puppet-api/v3/compile.markdown

```diff
@@ -36,7 +36,9 @@ The request body must look like:
   "transaction_uuid": "<uuid string>",
   "job_id": "<id string>",
   "options": { "capture_logs": <true/false>,
-               "log_level": <one of debug/info/warning/err>}
+               "log_level": <one of debug/info/warning/err>
+               "bolt": <true/false> Whether or not to attempt to load bolt
```

Comment:
nit: change to bolt_types to identify what we are doing to bolt (loading bolt types)